app/views/foreman_ansible/ansible_roles/_hostgroup_ansible_roles_button.erb

```diff
@@ -1,8 +1,9 @@
 <%=
   play_roles = if hostgroup.all_ansible_roles.empty?
-    "<a title='No Roles assigned' href=\"#\">#{_('Play Roles')}</a>".html_safe
+    "<a title='No Roles assigned' href=\"#\" disabled='disabled'>#{_('Run all Ansible roles')}</a>".html_safe
+    link_to _('Run all Ansible roles'), 'javascript:void(0);', disabled: true, title: 'No Roles assigned'
```

Comment:
don't we have 2 times the same link here now? while only the second will be in fact printed...